src/mlpack/methods/lmnn/constraints_impl.hpp

```diff
@@ -145,33 +210,50 @@ void Constraints<MetricType>::Impostors(arma::Mat<size_t>& outputMatrix,
 // Calculates k differently labeled nearest neighbors. The function
 // writes back calculated neighbors & distances to passed matrices.
 template<typename MetricType>
-void Constraints<MetricType>::Impostors(arma::Mat<size_t>& outputNeighbors,
+void Constraints<MetricType>::Impostors(arma::Mat<size_t>& outputMatrix,
                                         arma::mat& outputDistance,
                                         const arma::mat& dataset,
-                                        const arma::Row<size_t>& labels)
+                                        const arma::Row<size_t>& labels,
+                                        const arma::mat& diffTransform)
```

Comment:
I think the name `diffTransform` is misleading here, shouldn't it just be `transformation`?

Reply:
Ahh! forgot to change this one. Thanks for pointing that out! It's corrected now.